### DISCUSSION TITLE
powershell-editor-services

### DISCUSSION THREAD

**Discussion by railgadi** [mason-org/mason.nvim#262] [Q&A] [answered]
I am trying to use powershell-editor-services on ubuntu 20.04 (over NvChad). Installed it using MasonInstall. Below is my custom lspconfig.lua. All other LSP's are working perfectly, however I am unable to make powershell-editor-services work.

On another machine, I have installed coc-powershell and its working as expected.

Could you please suggest how to use powershell-editor-services using Mason?

`local on_attach = require("plugins.configs.lspconfig").on_attach
local capabilities = require("plugins.configs.lspconfig").capabilities

local lspconfig = require "lspconfig"

local servers = { "html", "cssls", "tsserver","pyright","powershell_es","gopls", }

for _, lsp in ipairs(servers) do 
    lspconfig[lsp].setup {
    on_attach = on_attach,
    capabilities = capabilities,
  } 
end

local bp = '~/.config/PowerShellEditorServices'
lspconfig.powershell_es.setup{
  bundle_path = bp,
}
`

**Comment by railgadi** [comment, marked as answer]
I was able to make it work. Kindly close the issue.

**Reply by dw72** [reply]
What you done?
